rama-tls/src/boring/client/connector_data.rs

```diff
@@ -244,11 +235,29 @@ impl TlsConnectorData {
         }
 
         trace!("boring connector: build SSL connector config");
-        let cfg = cfg_builder
+        let mut cfg = cfg_builder
             .build()
             .configure()
             .context("create ssl connector configuration")?;
 
+        if let Some(limit) = self.connect_config_input.record_size_limit {
+            trace!("boring connector: setting record size limit");
+            cfg.set_record_size_limit(limit).unwrap();
+        }
+
+        if let Some(schemes) = self
+            .connect_config_input
+            .delegated_credential_schemes
+            .as_ref()
+        {
+            trace!("boring connector: setting delegated credential schemes");
+            cfg.set_delegated_credential_schemes(schemes).unwrap();
+        }
+
+        // TODO seems like we always want to enable this if we don't do actual ECH?
```

Comment:
can you update the comment to clarify the TODO Part please, it's inclear to me what the todo is